yt/frontends/adaptahop/io.py

```diff
@@ -209,7 +209,7 @@ def _get_particle_positions(self):
         # Make sure halos are loaded in increasing halo_id order
         assert np.all(np.diff(offset_map[:, 0]) > 0)
 
-        # Cache particle positions as one do not expect a (very) large number of halos anyway
+        # Cache particle positions as one do not expect a large number of halos anyway
```

Comment:
Is this a flake8 change? 🙂

Reply:
well it's _one_ way to make the line go under 88 chars ! Those are all human edits, there may be room for arguments 🤓

Reply:
haha i'm fine with this change. 🙂